src/common/components/DataTable/FilterDropdown.tsx

```diff
@@ -178,11 +216,17 @@ export const FilterDropdown: FC<{
 
         {filters.map(filter => (
           <FilterCategory key={filter.attribute} className={openFilter === filter ? 'open' : ''}>
-            <div role='button' tabIndex={-1} className='category' onClick={() => toggleCategory(filter)}>
-              <span className={activeFilters.filter(a => a.attr === filter.attribute).length ? 'active' : ''}>
-                {filter.attributeLabel}
-              </span>
-              <FontAwesomeIcon icon='chevron-down' />
+            <div id='action-container'>
```

Comment:
Same here, id's should only be used when there is only 1 of something on a page.